src/annotator/test/sidebar-test.js

```diff
@@ -506,14 +506,22 @@ describe('Sidebar', () => {
       });
 
       it('notifies when sidebar changes expanded state', () => {
+        sinon.stub(sidebar, 'publish');
         sidebar.show();
         assert.calledOnce(layoutChangeHandlerSpy);
+        assert.calledOnce(sidebar.publish);
```

Comment:
I try to avoid using an `assert.isTrue` wrapper around sinon assertions because they're harder to debug that way when they fail. If you use `isTrue` (or `isFalse`), the failure won't give you the actual failure reason (that is, it will report that it expected `true` but got `false`), whereas if I use `calledOnceWith` directly here, a test failure will output the expected and the actual args list, which is more informative.